src/app/dao-coins/dao-coins.component.html

```diff
@@ -0,0 +1,264 @@
+<!-- Top Bar -->
+<div
+  class="global__top-bar  global__top-bar__height d-flex align-items-center fs-18px font-weight-bold pl-15px border-bottom border-color-grey"
+  *ngIf="!inTutorial"
```

Comment:
Don't know that this is necessary, won't break anything if it's here but might be worth cleaning up